bitrise.yml

```diff
@@ -12,6 +12,8 @@ workflows:
     - integration-test
     title: Runs tests
     steps:
+#    - git::https://github.com/matrangam/bitrise-step-swift-test.git@main: { } # Binary
```

Comment:
why commented?

Reply:
I left these in here for testing purposes. 
Each one of the steps illustrates a code path for the CLI executing the Swift Step example.